HW5/Task26.py

```diff
@@ -0,0 +1,16 @@
+# Задача 26: Напишите программу, которая на вход принимает два числа A и B, и возводит число А в целую степень B с помощью рекурсии.
+# A = 3; B = 5 -> 243 (3⁵)
+# A = 2; B = 3 -> 8
+
+def power(a, b):
+    if b == 0:
+        return 1
+    elif b == 1:
+        return a
+    else:
+        res = power(a, b-1) * a
+        return res
```

Comment:
Это можно совместить
```suggestion
        return power(a, b-1) * a
```